vita3k/gui/src/settings_dialog.cpp

```diff
@@ -385,13 +391,18 @@ void draw_settings_dialog(GuiState &gui, HostState &host) {
     if (ImGui::BeginTabItem("Emulator")) {
         ImGui::PopStyleColor();
         ImGui::Spacing();
+        ImGui::Checkbox("Disable At9 audio decoder", &config.disable_at9_decoder);
+        if (ImGui::IsItemHovered())
+            ImGui::SetTooltip("Disable At9 audio decoder can help some game to more progress.\nThis option is at the moment obligatory request for Gravity Rush/Daze");
```

Comment:
Enable the hack that disables AT9 audio decoder.\nIt is required to prevent the crash in certain games such as Gravity Rush or Daze for the time being.